client/state/jetpack-connect/reducer.js

```diff
@@ -47,7 +47,7 @@ const defaultAuthorizeState = {
 };
 
 function buildNoProtocolUrlObj( url, flowType ) {
-	const noProtocolUrl = url.replace( /.*?:\/\//g, '' );
+	const noProtocolUrl = url.replace( /.*?:\/\//g, '' ).replace( /\//g, '::' );
```

Comment:
if we are replacing the `/` with `::` we would need to change this name from "noProtocolUrl" to "slug" ...  I'm